### ISSUE TITLE
[Feature Request] Telegram chat room feature suggestion

### ISSUE THREAD

**Issue by sH1222J** [kotatogram/kotatogram-desktop#184]
### Is your feature request related to a problem?
A clear and concise description of what the problem is.

### Describe the solution you'd like

Conversation list and chat window are integrated.
Is it possible to separate it? (pop-up dialog)

### Additional context
![1](https://user-images.githubusercontent.com/28824568/123518384-9eca2b80-d6e0-11eb-858e-75d14ee044a1.png)

**Comment by ilya-fedin** [comment]
Not really, tdesktop code needs a big rewrite top be able to handle multiple main windows
